scripts/lib.sh

```diff
@@ -20,13 +30,34 @@ query-mac() {
     [[ $(uname -s) = 'Darwin' ]]
 }
 
-query_big_sur() {
+query-big-sur() {
     if require sw_vers && sw_vers -productVersion | grep -E "11\." >/dev/null; then
         return 0
     fi
     return 1
 }
 
+query-apple-m1() {
+    query-mac && [[ $(uname -m) = 'arm64' ]]
+}
+
+query-pyenv-version() {
+    local PYENV_VERSION
+    PYENV_VERSION=3.6.13
+    query-apple-m1 && PYENV_VERSION=3.8.10 || :
+    echo "${PYENV_VERSION}"
+}
```

Comment:
And here since we're returning the current pyenv version, I would use `get` instead of `query`